pipeline/scripts/download-electron-mirror.js

```diff
@@ -1,26 +1,51 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 const pkg = require('../../package.json');
-const download = require('electron-download');
-const unzipper = require('unzipper');
+const { downloadArtifact } = require('@electron/get');
+const extract = require('extract-zip');
 const fs = require('fs');
+const path = require('path');
 
-download(
-    {
-        version: pkg.dependencies.electron,
-        cache: 'drop/zips',
-    },
-    function (err, zipPath) {
-        // zipPath will be the path of the zip that it downloaded.
-        // If the zip was already cached it will skip
-        // downloading and call the cb with the cached zip path.
-        // If it wasn't cached it will download the zip and save
-        // it in the cache path.
-        if (err) {
-            console.log('Failed to download: ', err);
-        } else {
-            console.log('zipPath= ', zipPath);
-            fs.createReadStream(zipPath).pipe(unzipper.Extract({ path: 'drop/electron-local' }));
-        }
-    },
-);
+/*
+This script replaces existing electron & chromedriver modules
+with mirror dependencies specified by ELECTRON_MIRROR_VAR
+and ELECTRON_CUSTOM_DIR_VAR (see @electron/get).
```

Comment:
It might be nice to also note why we do this (something like "We use this to avoid bundling non-freely-redistributable media codecs in our release builds. The version of Electron published to npm includes these as part of Chromium; our release builds use a Microsoft-maintained build of Electron that removes those codecs.")

Reply:
Thanks for the suggested wording, have added it